tests/Integration/Behaviour/Features/Context/Domain/ShowcaseCardFeatureContext.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Tests\Integration\Behaviour\Features\Context\Domain;
+
+use PHPUnit_Framework_Assert;
+use PHPUnit_Framework_AssertionFailedError;
+use PrestaShop\PrestaShop\Core\Domain\ShowcaseCard\Command\CloseShowcaseCardCommand;
+use PrestaShop\PrestaShop\Core\Domain\ShowcaseCard\Exception\InvalidShowcaseCardNameException;
+use PrestaShop\PrestaShop\Core\Domain\ShowcaseCard\Exception\ShowcaseCardException;
+use PrestaShop\PrestaShop\Core\Domain\ShowcaseCard\Query\GetShowcaseCardIsClosed;
+use PrestaShopException;
+use Shop;
+use Symfony\Component\DependencyInjection\Exception\ServiceCircularReferenceException;
+use Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException;
+
+class ShowcaseCardFeatureContext extends AbstractDomainFeatureContext
+{
+    /**
+     * @Given single shop context is loaded
+     *
+     * @throws PrestaShopException
+     */
+    public function singleShopContextIsLoaded()
```

Comment:
We might need to reuse these steps in other scenarios. Maybe we can move these steps into `AbstractDomainFeatureContext` ?